ansible/roles/test/tasks/neighbour-mac-noptf.yml

```diff
@@ -0,0 +1,156 @@
+- block:
+  - setup:
+
+  - name: Get orchagent process information, make sure Orchagent is running before test 
+    shell: pgrep orchagent
+    register: pgrep_out
+
+  - debug: var=pgrep_out.stdout_lines
+  
+  - name: Verify that orchagent process is running
+    assert: { that: "{{ pgrep_out.stdout_lines | length }} > 0"}
+
+  - name: Gathering minigraph facts about the device
+    minigraph_facts: host={{ inventory_hostname }}
+    connection: local
+    become: no
+    tags: always
+
+  - set_fact:
+      testname: "mgtest"
+      out_dir: "/tmp"
+      run_dir: "/tmp"
+      v4_mac1: "08:bc:27:af:cc:45"
+      v4_mac2: "08:bc:27:af:cc:47"
+      v6_mac1: "08:bc:27:af:cc:65"
+      v6_mac2: "08:bc:27:af:cc:67"
+
+  - name: init loganalyzer for later syslog analysis
+    include: roles/test/files/tools/loganalyzer/loganalyzer_init.yml
+
+  - name: gather DUT arp table
+    switch_arptable: 
+
+  # - debug: var=arptable
+
+  - fail: msg="cannot find a IPv4 and IPv6 neighbor to change mac address"
+    when: ( arptable.v4 | length == 0 ) or ( arptable.v6 | length == 0 )
```

Comment:
we do not need to fail if there is no neighbor, we can just first create a neigbhor entry.